tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/logwatch/ModelNodeLogWatcher.java

```diff
@@ -0,0 +1,74 @@
+package org.jboss.eap.qe.microprofile.tooling.server.logwatch;
+
+import org.jboss.dmr.ModelNode;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
```

Comment:
@mnovak1 you have run into some issues using Creaper with in-container Arquillian mode. However there are tests to be executed in such mode. Any thoughts?

Reply:
@istraka in case it will be in-container test then the whole creaper core (with wildfly-cli and wildfly-controller-client must be added)

Reply:
yep, we will need to have a deployment which would contain all dependencies necessary. So I am thinking about removing all non-client tests.

Reply:
> in case it will be in-container test then the whole creaper core (with wildfly-cli and wildfly-controller-client must be added)

@mnovak1, @honza-kasik could a test for in-container usage be added once this gets merged?

Reply:
@fabiobrz I think that it would be the best solution, when @mnovak1 has actually experience doing this.

Reply:
@fabiobrz I'm not sure when I get to it. Maybe tomorrow so anyone is free to try :-)